src/components/PoolOverview.tsx

```diff
@@ -148,6 +149,10 @@ export default function PoolOverview({
   )
 }
 
-function Tag(props: { children?: React.ReactNode }) {
-  return <span className="tag" {...props} />
+function Tag(props: {
+  children?: React.ReactNode
+  kind?: "warning" | "error"
+}) {
+  const { kind = "warning", ...tagProps } = props
+  return <span className={`tag ${kind}`} {...tagProps} />
```

Comment:
nit: prefer to use `className={classnames("tag", kind)}` rather than doing manual interpolation